examples/local/azure-resource-lab-chart/values.yaml

```diff
@@ -0,0 +1,11 @@
+clusterName: "test-cluster"
+commonDomain: "azure.gigantic.io"
+sshUser: "test-user"
+sshPublicKey: "ssh-rsa XXXXXXXXXXXXXXXXXXXXXXXXX"
+encryptionKey: "QitRZGlWeW5WOFo2YmdvMVRwQUQ2UWoxRHZSVEF4MmovajlFb05sT1AzOD0="
```

Comment:
Added to the readme it's for encrypting the etcd storage. I added it as it will be needed once the k8scloudconfig version is unpinned.

See https://github.com/giantswarm/k8scloudconfig/blob/30de6e46e3cd97ad7a6da9005fa00f9dc785b179/v_0_1_0/master_template.go#L2038